core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerClient.java

```diff
@@ -76,7 +78,21 @@ public final class CircuitBreakerClient extends AbstractCircuitBreakerClient<Htt
     newDecorator(CircuitBreakerMapping mapping, CircuitBreakerRule rule) {
         requireNonNull(mapping, "mapping");
         requireNonNull(rule, "rule");
-        return delegate -> new CircuitBreakerClient(delegate, mapping, rule);
+        return newDecorator(CircuitBreakerClientHandler.of(mapping), rule);
+    }
+
+    /**
+     * Creates a new decorator with the specified {@link CircuitBreakerClientHandler} and
+     * {@link CircuitBreakerRule}.
+     *
+     * @see CircuitBreakerClientHandler
```

Comment:
ditto. If you want to add more description about the parameter, how about using `@param` and adding description to it?